mkdocs.yml

```diff
@@ -30,6 +30,8 @@ nav:
   - '_Serial Tutorial': serial-tutorial.md
   - '_Parallel Tutorial': parallel-tutorial.md
   - '_Code Overview': code-overview.md
+  - '_Tesla Notes': tesla-notes.md
```

Comment:
Since these are hidden perhaps we should add links to the getting-started.md page to reproduce the menu items shown here with underscores.

Reply:
Oh I like that idea.  I'm going to add a theory notes section there.